server/series/api.py

```diff
@@ -265,6 +279,101 @@ def revoke_series_invitation(
     return 200, invitation
 
 
+@router.get(
```

Comment:
Kept this as GET since it does not require csrf verification. Also since `csrf_exempt` is not working for the API endpoints in series. My guess is its being enforced at some other level as well.